diff --git a/CMake/HPHPFindLibs.cmake b/CMake/HPHPFindLibs.cmake
index 8f30baf..d32784e 100644
--- a/CMake/HPHPFindLibs.cmake
+++ b/CMake/HPHPFindLibs.cmake
@@ -56,7 +56,7 @@ find_package(LibEvent REQUIRED)
 include_directories(${LIBEVENT_INCLUDE_DIR})
 
 set(CMAKE_REQUIRED_LIBRARIES "${LIBEVENT_LIB}")
-CHECK_FUNCTION_EXISTS("evhttp_bind_socket_with_fd" HAVE_CUSTOM_LIBEVENT)
+CHECK_FUNCTION_EXISTS("evhttp_bind_socket_with_handle" HAVE_CUSTOM_LIBEVENT)
 if (NOT HAVE_CUSTOM_LIBEVENT)
 	unset(HAVE_CUSTOM_LIBEVENT CACHE)
 	unset(LIBEVENT_INCLUDE_DIR CACHE)
